conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,118 @@
+what reports should contain. If a report doesn't contain enough information, the
+committee will obtain all relevant data before acting. The committee is
+empowered to act on the Steering Council’s behalf in contacting any individuals
+involved to get a more complete account of events.
+
+The committee will then review the incident and determine, to the best of their
+ability:
+
+* What happened
+* Whether this event constitutes a Code of Conduct violation
+* Who, if anyone, was responsible for the violation
+* Whether this is an ongoing situation, and there is a threat to anyone's
+  physical safety
+
+This information will be collected in writing, and whenever possible the group's
+deliberations will be recorded and retained (i.e. chat transcripts, email
+discussions, recorded voice conversations, etc).
+
+It is important to retain an archive of all activities of this committe to
+ensure consistency in behavior and provide institutional memory for the
+project.
+
+The Code of Conduct Committee should aim to have a resolution agreed upon within
+one week. In the event that a resolution can't be determined in that time, the
+committee will respond to the reporter(s) with an update and projected timeline
+for resolution.
+
+
+## Acting Unilaterally
+
+If the act is ongoing, or involves a threat to anyone's safety (e.g. threats of
+violence), any committee member may act immediately (before reaching consensus)
+to address the situation. In ongoing situations, any member may decide to employ
+any of the tools available to the committee, including bans and blocks.
+
+If the incident involves physical danger, any member of the committee may -- and
+should -- act unilaterally to protect the safety of those involved. This can
+include contacting law enforcement (or other local personnel) and speaking on
+behalf of the Jupyter Steering Council.
+
+In situations where an individual committee member acts unilaterally, they must
+report their actions to the committee for review within 24 hours.
+
+
+## Resolutions
+
+The committee must agree on a resolution by consensus. If the group cannot reach
+consensus and deadlocks for over a week, the group will turn the matter over to
+the Steering Council for resolution.
+
+Possible responses may include:
+
+* Taking no further action (if we determine no violation occurred).
+* A private reprimand from the committee to the individual(s) involved. In this
+  case, the group chair will deliver that reprimand to the individual(s) over
+  email, cc'ing the group.
+* A public reprimand. In this case, the committee chair will deliver that
+  reprimand in the same venue that the violation occurred (i.e. in the gitter
+  chat room for a gitter chat room violation; email for an email violation,
+  etc.). The group may choose to publish this message elsewhere for
+  documentation purposes.
+* An imposed vacation (i.e. asking someone to "take a week off" from a mailing
```

Comment:
I think "vacation" connotes a reward or wrist slap for subpar behavior. 

I would prefer:  A "mutually agreed upon hiatus" where the committee asks the individual to temporarily refrain from community participation. If the individual chooses not to take a temporary break voluntarily, the committee may issue a "mandatory cooling off period".